umbraco-heartcore/getting-started/webhooks.md

```diff
@@ -1,48 +1,77 @@
 # Webhooks
 
-In this article you can learn more about how to use Webhooks, and how to set them up.
+Webhooks give you the ability to send information about events in Heartcore to external systems. They work by sending an HTTP POST request to a configured endpoint, with information about the event in the request body.
 
-Webhooks from the backoffice works the same way as the webhooks from the portal. It gives you an option to hook into different actions in order to post information about the action elsewhere.
+## Uses
 
-## Usage
+Webhooks can keep other systems in-sync with content from Heartcore without having to poll it for changes.
 
-An example of when to use webhooks would be if you have a website created with a static page builder.
+One common use-case is building static websites. By adding a webhook, you can inform your chosen static site builder when content has changed so that it can re-generate static assets on-demand.
```

Comment:
```suggestion
One common use case is building static websites. By adding a webhook, you can inform your chosen static site builder when content changes so that it can re-generate static assets on demand.
```